xla/service/gpu/matmul_utils.h

```diff
@@ -32,11 +32,20 @@ limitations under the License.
 #include "xla/types.h"
 #include "xla/xla_data.pb.h"
 
+#if TENSORFLOW_USE_ROCM
+#include "rocm/rocm_config.h"
+#endif
+
 #if GOOGLE_CUDA
 #include "xla/stream_executor/cuda/cuda_blas_lt.h"
 #include "xla/stream_executor/scratch_allocator.h"
 #endif  // GOOGLE_CUDA
 
+#if TENSORFLOW_USE_ROCM && TF_HIPBLASLT
```

Comment:
Can you use #elif here?
Just to make it clear that we don't have a duplicate include.

Reply:
Done. Thanks a lot for the check.

https://github.com/openxla/xla/pull/3953/files#diff-98376f6c713755983f4458b3660033af9384bf588a8aea20e226624d7fcefeeeR55-R59

It seems ```#if GOOGLE_CUDA``` doesn't work in cuda_blas_lt.cc and failed to compile at cuda side, I am not familar with cuda side, wonder do you have any idea why it doesn't work there? Thanks in advance!

Reply:
It is possible that GOOGLE_CUDA is not defined. Ideally, we would not need GOOGLE_CUDA macros inside the stream_exector/cuda directory. So far, we only need it in tests. Can you maybe find a different way to share code and create a shared target outside the cuda directory? I have not thought about this myself, so maybe it is not possible. Your current usage of cuda_blas_lt.cc as header is definitely not ok though.